app/bundles/ConfigBundle/Model/SysinfoModel.php

```diff
@@ -5,11 +5,12 @@
 use Doctrine\DBAL\Connection;
 use Mautic\CoreBundle\Helper\CoreParametersHelper;
 use Mautic\CoreBundle\Helper\PathsHelper;
+use Mautic\CoreBundle\Model\MauticModelInterface;
 use Mautic\InstallBundle\Configurator\Step\CheckStep;
 use Mautic\InstallBundle\Install\InstallService;
 use Symfony\Contracts\Translation\TranslatorInterface;
 
-class SysinfoModel
+class SysinfoModel implements MauticModelInterface
```

Comment:
As the `\Mautic\CoreBundle\Factory\ModelFactory::getModel` now returns a `MauticModelInterface` it's necessary that all models implement this interface. Also in the future will help with autowiring.

Reply:
I shouldn't have called this class a model probably. It's just a service that have nothing in common with the classic Mautic models that are used work with entities, permissions and so on. I called it a model because it manages the data needed for the system info view. But otherwise doesn't need any special attention that the classic Mautic models need.

Reply:
Removed the interface. Now these models are injected into controllers.